source/language/delays.rst

```diff
@@ -241,9 +221,7 @@ for the entire duration of some other gate.
 A multi-qubit ``delay`` instruction is *not* equivalent to multiple single-qubit
 ``delay`` instructions. Instead a multi-qubit delay acts as a synchronization
 point on the qubits, where the delay begins from the latest non-idle
-time across all qubits, and ends simultaneously across all qubits. For
-this reason, a ``barrier`` instruction is exactly equivalent to a ``delay`` of a length zero
-on the qubits involved.
```

Comment:
Why did you drop this comment about being equivalent to a `delay` of length zero?

Reply:
I moved it to this section https://github.com/Qiskit/openqasm/blob/a18f2cb9de95408601a0a1abf9c3ae1caccd5497/source/language/delays.rst#barrier-instruction, where `barrier`'s are actually introduced. The language is derived from the discussion on the bottom of pp. 26 of https://arxiv.org/pdf/2104.14722.pdf.